.github/workflows/cwag-workflow.yml

```diff
@@ -156,6 +156,13 @@ jobs:
         run: |
           cd ${MAGMA_ROOT}/cwf/gateway/docker
           docker-compose --file docker-compose.yml --file docker-compose.override.yml build --parallel
+      - name: Retry docker compose on failure
+        if: failure()
```

Comment:
Use this instead: https://github.community/t/how-to-retry-a-failed-step-in-github-actions-workflow/125880/2
If you just use failure(), it will not be specific to the docker compose step

Reply:
thank you!